.github/workflows/ci.yml

```diff
@@ -37,22 +36,155 @@ jobs:
     - name: Build
       run: make build
 
-    - name: Lint
-      uses: golangci/golangci-lint-action@v1
-      with:
-        # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-        version: v1.27
-      
     - name: Test
       run: make test
 
     - uses: shogo82148/actions-goveralls@v1
       with:
         path-to-profile: cover.out
 
+    - uses: actions/upload-artifact@v2
+      with:
+        name: helm-operator
+        path: bin/helm-operator
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Lint
+      uses: golangci/golangci-lint-action@v1
+      with:
+        version: v1.27
+
+  e2e:
```

Comment:
Do you not agree in do all tests in Go? 
So, why not impl the e2e tests such as : https://github.com/operator-framework/operator-sdk/tree/master/test/e2e-new?

Reply:
we spoke about and the answer is yes.